debezium-core/src/main/java/io/debezium/config/Configuration.java

```diff
@@ -2097,4 +2097,47 @@ default void forEachMatchingFieldName(Pattern regex, BiConsumer<String, String>
     default <T> void forEach(BiConsumer<String, String> function) {
         this.asMap().forEach(function);
     }
+
+    /**
+     * Returns the string config value from newProperty config field if it's set or its default value when it's not
+     * set/null.If both are null it returns the value of the oldProperty config field, or its default value when it's
+     * null.
+     * This fallback only works for newProperty fields that have a null / not-set default value!
+     *
+     * @param newProperty the new property config field
+     * @param oldProperty the old / fallback property config field
+     * @return the evaluated value
+     */
+    default String getFallbackStringProperty(Field newProperty, Field oldProperty) {
+        return getString(
```

Comment:
As I think about that should not the warning be emitted when either old name or both new and old names are present?

Reply:
Not just silently using the documented (newer) version when both are there? I could add a warning but might be quite irrelevant, wdyt?

Reply:
I'd rather have a warning - just in case there are two persons working with connector config and each updates different param. In this case it is one more safety to avoid config inconsistency

Reply:
yep. will do that

Reply:
Done. And also changed the code to call the static method instead of the duplicated code.